spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/source/TestDataSourceOptions.java

```diff
@@ -407,13 +409,13 @@ public void testExtraSnapshotMetadata() throws IOException {
 
     Table table = tables.load(tableLocation);
 
-    Assert.assertTrue(table.currentSnapshot().summary().get("extra-key").equals("someValue"));
-    Assert.assertTrue(table.currentSnapshot().summary().get("another-key").equals("anotherValue"));
+    assertThat(table.currentSnapshot().summary().get("extra-key")).isEqualTo("someValue");
```

Comment:
in such cases it's usually better to do something like
```
assertThat(table.currentSnapshot().summary().containsEntry("extra-key", "someValue).containsEntry("another-key", "anotherValue")
```
because if the assertion ever fails, the full map content will be printed, which makes debugging much easier